SpatialGDK/Source/SpatialGDK/Public/Schema/RPCPayload.h

```diff
@@ -16,9 +16,10 @@ struct RPCPayload
 {
 	RPCPayload() = delete;
 
-	RPCPayload(uint32 InOffset, uint32 InIndex, TArray<uint8>&& Data, TraceKey InTraceKey = InvalidTraceKey)
+	RPCPayload(uint32 InOffset, uint32 InIndex, uint64 InId, TArray<uint8>&& Data, TraceKey InTraceKey = InvalidTraceKey)
```

Comment:
Shouldn't this be an optional? Or at least have some indication if it's actual set?